relayer-cli/src/commands/tx/client.rs

```diff
@@ -23,6 +24,18 @@ pub struct TxCreateClientCmd {
 
     #[clap(required = true, help = "identifier of the source chain")]
     src_chain_id: ChainId,
+
+    // FIXME: provide option doc
```

Comment:
```suggestion
    /// Override the default clock drift specified in the configuration.
    /// The clock drift is a correction parameter dealing with only approximately synchronized clocks.
    /// The local clock should always be ahead of timestamps from the blockchain; this
    /// is the maximum amount that the local clock may drift behind a timestamp from the
    /// blockchain.
```

Reply:
Right, let's do that!

Reply:
The long docs are applied in 6197b2dbc1007b35c789554306d783530faa1693, so you can check how it looks like with `--help`.